### ISSUE TITLE
Feature Request - Suppress MD033 Violations

### ISSUE THREAD

**Issue by elibroftw** [DavidAnson/markdownlint#467]
I use HTML in my project's `README.md` for things like links with images and images in the header. I would like a way to suppress the warnings the linter gives me.
For example
```html
<a href=https://youtu.be/5xwHkLPgvtQ>
  <img width=75% src="https://img.youtube.com/vi/5xwHkLPgvtQ/maxresdefault.jpg" alt="demo YouTube video thumbnail" align="center"/>
</a>
```
Gives me inline HTML errors but me using an Image inside an a tag should not give any such violations.
So the best thing would be to not spout out violations for HTML where there is an `img` tag nested within another tag.

**Comment by DavidAnson** [comment]
This library supports inline suppression as shown here: https://github.com/DavidAnson/markdownlint#configuration

You're probably using this library through a tool; they support additional ways to suppress (for example, with a file): https://github.com/DavidAnson/markdownlint#related

**Comment by elibroftw** [comment]
Works. Thanks
